src/unparse.c

```diff
@@ -343,6 +343,10 @@ void output_constant(known_cst c)
       output_quoted_cs(ddecl->schars);
       output("\"");
     }
+  else if (type_pointer(t))
```

Comment:
This is also suspicious (and violates the "Requires:" clause on the function signature). I suspect it should not be needed with the cval_cast change.

Reply:
Currently only char[] and number is supported.  To support null pointer as a constant, I added output_null_constant() function